lift.toml

```diff
@@ -7,9 +7,9 @@ description = "Ship your interpreted executables using science."
 [[lift.interpreters]]
 id = "cpython"
 provider = "PythonBuildStandalone"
-release = "20230826"
-version = "3.11.5"
-# By default science ships as a "thin" scie that fetches CPython 3.11 on first run.
+release = "20231002"
+version = "3.12.0"
```

Comment:
Not that it matters for this PR - but worth noting to anyone who uses Python 3.12 later.

https://github.com/indygreg/python-build-standalone/releases/tag/20231002

> CPython 3.12.0 now supported. These distributions should be considered beta quality since it is their initial release and typically there are both upstream bugs and bugs in our packaging in the initial release. So exercise caution before deploying to production.

Reply:
Yeah. I wanted to get on the test train early - someone has to. CI (`nox`) fully dogfoods; so I'm confident `science` is still production ready for the next release.